requirements.txt

```diff
@@ -2,7 +2,7 @@
 -r requirements_production.txt
 
 # Requirements for development and tests in alphabetical order
-coverage>=4.0a5,<4.1.0
+coverage==4.0a5
```

Comment:
This breaks travis. It seams that the 4.0a6 needs allow-external.

I am not sure if this was a temporal problem, If you like, you can update the coverage version with a seperate pull request.